applications/empire_application/python_scripts/empire_mapper.py

```diff
@@ -2,19 +2,17 @@
 # import libraries
 from KratosMultiphysics import *
 from KratosMultiphysics.EmpireApplication import *
-from KratosMultiphysics.IncompressibleFluidApplication import *
 import ctypes as ctp
 import os
 
 CheckForPreviousImport()
 
+path_of_the_application =  os.path.dirname((os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe()))))) # script directory
+# print(path_of_the_appli)
 # Import the mapper library
-# This is pre compiled and is put in the external library. Also a header file is put in the header file folder to give the 
-# users the syntax for the mapping routines. 
-#libMapper = ctp.cdll.LoadLibrary("../custom_external_libraries/libEMPIRE_MapperLib.so")
-libMapper = ctp.cdll.LoadLibrary("/home/annab/software/kratos/applications/empire_application/custom_external_libraries/libEMPIRE_MapperLib.so")
-
-#libMapper = ctp.cdll.LoadLibrary("libEMPIRE_MapperLib.so")
+# This is pre compiled and is put in the external library. Also a header file is put in the header file folder to give the
+# users the syntax for the mapping routines.
+libMapper = ctp.cdll.LoadLibrary(path_of_the_application+"/custom_external_libraries/libEMPIRE_MapperLib.so")

```

Comment:
As far as I know this library is compile by @adityaghantasala and put there.
Since I am only user and not developer I cannot say much about this.

@adityaghantasala can you please address Jordis questions?